modules/market/src/main/java/com/opengamma/strata/market/curve/Curve.java

```diff
@@ -147,4 +148,33 @@ public default CurrencyParameterSensitivity createParameterSensitivity(Currency
     return CurrencyParameterSensitivity.of(getName(), paramMeta, currency, sensitivities);
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Obtains a list of underlying curves. 
+   * <p>
+   * In most cases, the number of underlying curves is 1. Thus a list of this curve is returned.
+   * 
+   * @return the underlying curves
+   */
+  public default ImmutableList<Curve> split() {
+    return ImmutableList.of(this);
+  }
+
+  /**
+   * Replaces an underlying curve by a new curve.
+   * <p>
+   * {@code curveIndex} must be coherent to the index of the list in {@code split()}.
+   * If {@code curveIndex} is outside the range, {@code this} instance is returned.
+   * 
+   * @param curveIndex  the curve index
+   * @param curve  the new split curve
+   * @return the new curve
+   */
+  public default Curve withUnderlyingCurve(int curveIndex, Curve curve) {
```

Comment:
Does these methods not need to be implemented anywhere else? Like `AddFixedCurve`?